types/module/module.go

```diff
@@ -260,10 +260,6 @@ func (m *Manager) RegisterRoutes(router sdk.Router, queryRouter sdk.QueryRouter)
 func (m *Manager) InitGenesis(ctx sdk.Context, cdc codec.JSONMarshaler, genesisData map[string]json.RawMessage) abci.ResponseInitChain {
 	var validatorUpdates []abci.ValidatorUpdate
 	for _, moduleName := range m.OrderInitGenesis {
-		if genesisData[moduleName] == nil {
```

Comment:
Removed this check since I needed transfer module to just bindport on InitChain even though there isn't yet any GenesisState for transfer module. I can add this check back if people want me to create dummy genesis state.

Either way, think it's important for modules to be able to run code in InitChain even if they don't have data in Genesis file

Reply:
I agree although it would be good to check with whomever wrote this logic originally.

Reply:
This is causing a number of test failures afaik

Reply:
cc: @alexanderbez @rigelrozanski

Reply:
Yes this would have to be the place that's causing Sim and gaia failures. Looking to see if there's a quick fix

Reply:
Decided to add this condition back since it was causing test failures. Created a simple default genesis state for transfer module to circumvent this for now